Security/main_widget_Security_Card.yaml

```diff
@@ -234,21 +234,32 @@ slots:
-                                            style:
-                                              font-weight: 800
-                                              fontSize: 14px
-                                              margin-top: -7px
-                                            title: "=loop.windowMember.metadata.uiSemantics ? loop.windowMember.metadata.uiSemantics.config.equipment + loop.windowMember.metadata.uiSemantics.config.preposition + loop.windowMember.metadata.uiSemantics.config.location : loop.windowMember.label"
+                                            accordionList: true
+                                            fetchMetadata: semantics,widgetOrder,uiSemantics
+                                            filter: "(loop.windowFilter.name == loop.windowMember.metadata.semantics.config.isPointOf) ? true : false"
+                                            for: windowFilter
+                                            fragment: true
+                                            itemTags: Window
+                                            sourceType: itemsWithTags
                                           slots:
                                             default:
-                                              - component: f7-chip
+                                              - component: f7-list-item
                                                 config:
-                                                  bgColor: transparent
-                                                  iconColor: '=items[loop.windowMember.name].state == "CLOSED" ? "gray" : "red"'
-                                                  iconMaterial: sensor_window
-                                                  iconSize: 20px
+                                                  style:
+                                                    font-weight: 800
+                                                    fontSize: 14px
+                                                    margin-top: -7px
+                                                  title: "=loop.windowMember.metadata.uiSemantics ? loop.windowMember.metadata.uiSemantics.config.equipment + loop.windowMember.metadata.uiSemantics.config.preposition + loop.windowMember.metadata.uiSemantics.config.location : loop.windowMember.label"
+                                                slots:
+                                                  default:
+                                                    - component: f7-chip
+                                                      config:
+                                                        bgColor: transparent
+                                                        iconColor: '=items[loop.windowMember.name].state == "CLOSED" ? "gray" : "red"'
+                                                        iconMaterial: sensor_window
+                                                        iconSize: 20px
                     inner-end:
```

Comment:
Can you please elaborate why yo think those changes are needed. I don‘t see it.

Reply:
So it is YOUR special use case, but not a general issue.
In that case, I am not going to merge this.
What is the purpose on having that information on the Thermostat but not to be used for indicating an open window?

Reply:
It is used - but without this change set the same information is displayed twice.
1. The real window contact
2. The internal thermostat state - that is derived from all windows contacts in that room
In the widget I would like to show only the information taken from window contacts, not just any "OpenState" item found in the model.

The question is: What the intention of the meta model for an element `homematic:HmIP-STHD:<xxxxx>:<yyyy>:1#WINDOW_STATE`?

Is the correct category
* Window
or
* Heating

I believe it is `Window`.

Reply:
Well this is not really clear. Some thermostats have a window open indication, not being derived from a real window contact. Therefore I would tend to tag it as „Control, Heating“, as that is what it does.